openapi-cli/src/main/java/io/ballerina/openapi/generators/schema/TypeGeneratorUtils.java

```diff
@@ -163,8 +163,13 @@ public static List<Node> addRecordFields(List<String> required, Set<Map.Entry<St
                     if (field.getValue().getDefault() != null) {
                         Token defaultValue;
                         if ((field.getValue().getType()).equals(STRING)) {
-                            defaultValue = AbstractNodeFactory.createIdentifierToken("\"" +
-                                    field.getValue().getDefault().toString() + "\"");
+                            if (field.getValue().getDefault().toString().equals("\"")) {
```

Comment:
Don't we need to trim this default value, if in case we have something like this with extra space ` default: ' "  '` ?

Reply:
Trim the white spaces